app/src/main/java/com/eveningoutpost/dexdrip/Models/Calibration.java

```diff
@@ -906,9 +912,15 @@ public static void adjustRecentBgReadings(int adjustCount) {
                             bgReading.filtered_calculated_value = new_calculated_value;
                         }
                         bgReading.calculated_value = new_calculated_value;
+                        
+                        Log.v(TAG, "Previous value: " + oldYValue + ", New value: " + bgReading.calculated_value);
```

Comment:
It looks like this log was used for development. Can we remove it before we merge? Logging in each step of the iteration will pollute our logs.

Reply:
I'm on-board with removing it if that's the consensus of the maintainers.
Personally I like it, because to actually get these logs you have to specifically add calibration logging, so 99% of people would never see it.

Reply:
I agree with @boc-the-git but suggest to print two decimal places at max. Please try to fit into one line.

Reply:
Please use `Log.v(TAG, String.format("Previous value: %.2f, New value: %.2f", oldYValue, bgReading.calculated_value));`